.vscode/settings.json

```diff
@@ -24,4 +24,7 @@
         "thecakelab",
         "Xihe"
     ],
+    "conventionalCommits.scopes": [
```

Comment:
What's the purpose of this setting?

Reply:
https://www.conventionalcommits.org/en/v1.0.0/

It's just a setting for Conventional Commit VSCode extension to pick up scopes defined in the workspace

Reply:
Interesting. Do you find it helpful in your typical git workflow?

Reply:
I just find it more useful for reviewers as they can narrow down the scope of what the commit's changing

Reply:
Sounds good. Let me see if we can make this tool part of the standard workflow for this project.